File: src/main/java/se/ifmo/pepe/dto/PointDataDTO.java
package se.ifmo.pepe.dto;


import io.swagger.annotations.ApiModelProperty;
import lombok.Data;

import javax.persistence.*;


@Data
public class PointDataDTO {
    @ApiModelProperty(position = 0)
    private Double x;
    @ApiModelProperty(position = 1)
    private Double y;
    @ApiModelProperty(position = 2)
    private Double r;
}

File: src/main/java/se/ifmo/pepe/dto/PointDataDTO.kt
package se.ifmo.pepe.dto

import io.swagger.annotations.ApiModelProperty

data class PointDataDTO(
    @ApiModelProperty(position = 0)
    val x: Double,
    @ApiModelProperty(position = 1)
    val y: Double,
    @ApiModelProperty(position = 2)
    val r: Double
)
